bin/impl/fluo_deploy/main.py

```diff
@@ -399,20 +391,6 @@ def main():
 
     response = raw_input("Do you want to continue? (y/n) ")
     if response == "y":
-      if config.subnet_id():
```

Comment:
Yes. The IPs only stick around if you allocate them and assign them by hand.  I also verified it that they are released.